### ISSUE TITLE
TS 6.1: hwclock not found

### ISSUE THREAD

**Issue by mcsn747** [Thinstation/thinstation#457]
Hello!
I accidentally discovered that the cron task hwclock doesn't executing
```
journalctl | grep hwclock
Sep 16 13:01:00 ts_001d60054a63 crond[5936]: /bin/sh: hwclock: not found
Sep 16 13:01:00 ts_001d60054a63 crond[5936]: /bin/sh: hwclock: not found
Sep 16 14:01:00 ts_001d60054a63 crond[5936]: /bin/sh: hwclock: not found

```
You must set full path to hwclock.

**Comment by Doncuppjr** [comment]
I'm not able to reproduce this error. All of my versions of busybox have the hwclock applet. Do you have a custom busybox?

**Comment by mcsn747** [comment]
I'm cloned your repo a week ago, and hwclock is in sbin folder.
crontab -l
1 * * * * /etc/init.d/ntpd restart
1 * * * * hwclock -w
*/1 * * * * /bin/check_x11vnc
0  * * * * /bin/checklogs
All other entries include full path to executable. You must set full path to hwclock.

**Comment by Thinstation** [comment]
Should be fixed in 6.2
